release/stage-release.sh

```diff
@@ -51,7 +58,7 @@ mkdir -p "${changelog_dir}"
 # TODO: Find a nicer way to do this.
 [ "${patch}" != "0" ] && printf "\n#" >>"${changelog_path}"
 
-releaser changelog compliantkubernetes-apps "${full_version}" >>"${changelog_path}"
+releaser changelog compliantkubernetes-apps "${full_version}" --github-token="${CK8S_GITHUB_TOKEN}" >>"${changelog_path}"
```

Comment:
No need to set the command line flag if you use the environment variable.

```suggestion
releaser changelog compliantkubernetes-apps "${full_version}" >>"${changelog_path}"
```